src/class-rest.php

```diff
@@ -121,6 +127,12 @@ public function get_callback( array $object ): array {
 	public function get_rendered_meta(): string {
 		ob_start();
 		$this->parsely->insert_page_header_metadata();
-		return ob_get_clean();
+		$meta = ob_get_clean();
+
+		if ( false === $meta ) {
+			return '';
+		}
+
+		return $meta;
```

Comment:
This change seems a little redundant, as output buffering is turned on two lines above.